js/navbar.js

```diff
@@ -1,5 +1,18 @@
 const hamburger = document.querySelector('.hamburger');
 const navMenu = document.querySelector('.nav-menu');
+const checkURL = window.location.href;
+const stagingURL = "https://staging-welcome.realdevsquad.com/";
+const productionURL = "https://welcome.realdevsquad.com/";
+
+function checkWindowURL() {
```

Comment:
Could you use [DOMContentLoaded](https://developer.mozilla.org/en-US/docs/Web/API/Window/DOMContentLoaded_event) event here. And put the code on top of the file.